### ISSUE TITLE
Malewarebytes says ransomware now quarntined 

### ISSUE THREAD

**Issue by ubiwabi** [subhra74/xdm#285]
This is the second time using the latest version of being set as ransomeware. I unistalled it and then reinstalled it and same thing happened.
I will let malewarebytes know 

(end)
Malwarebytes
www.malwarebytes.com

-Log Details-
Protection Event Date: 8/3/20
Protection Event Time: 10:10 PM
Log File: aa4754aa-d5f7-11ea-ab52-c8d9d2d2b2a1.json

-Software Information-
Version: 4.1.2.73
Components Version: 1.0.990
Update Package Version: 1.0.27903
License: Premium

-System Information-
OS: Windows 10 (Build 18362.959)
CPU: x64
File System: NTFS
User: System

-Ransomware Details-
File: 1
Malware.Ransom.Agent.Generic, C:\Program Files (x86)\XDM\java-runtime\bin\javaw.exe, Blocked, 0, 392685, 0.0.0

(end)

**PLEASE DO NOT JUST SAY "It does not work, or something not working etc." Provide enough relevent details so that the issue can be analyzed and reproduced easily**

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**please complete the following information:**
 - OS: [e.g. Linux/Windows]
 - Browser [e.g. chrome, Firefox]
 - XDM addon Version [e.g. 2.0]
 - XDM Version [e.g. 7.2.0]

**Generated log using below method**
 - https://github.com/subhra74/xdm/wiki/Generate-log-for-troubleshooting

**Additional context**

**Comment by subhra74** [comment]
Yes. Please report to malware bytes for the false positive. Previously also I faced this problem with many antivirus products and had to write to them to whitelist XDM from their list. This is really annoying

**Comment by ubiwabi** [comment]
Hello
I did that and added to allow list
Thank you
